app/src/main/java/test1/daihocbachkhoa/tranphuc1995/assignment1/MainActivity.java

```diff
@@ -0,0 +1,91 @@
+package test1.daihocbachkhoa.tranphuc1995.assignment1;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import java.util.ArrayList;
+
+public class MainActivity extends AppCompatActivity {
+    private RecyclerView recyclerView;
+    private RecyclerViewAdapter adapter;
+    private ArrayList<Beverage> arlDrink = new ArrayList<>();
+    private Button btnConfirmOrder;
+    private StringBuilder orderBeverages = new StringBuilder();
+    private EditText edtNumberOfOrder;
+    private ArrayList<String> arlNumberOfOrder = new ArrayList<>();
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        createData();
+        init();
+        confirmOrder();
+    }
+
+    private void init(){
+        recyclerView = (RecyclerView)findViewById(R.id.recyclerViewList);
+        btnConfirmOrder = (Button)findViewById(R.id.button_Confirm);
+        edtNumberOfOrder = (EditText)findViewById(R.id.editText_NumberOfOrder);
+        arlDrink = createData();
+        adapter = new RecyclerViewAdapter(arlDrink);
+        LinearLayoutManager layoutManager = new LinearLayoutManager(getApplicationContext());
+        layoutManager.setOrientation(LinearLayoutManager.VERTICAL);
+        recyclerView.setLayoutManager(layoutManager);
+        recyclerView.setAdapter(adapter);
+
+    }
+
+    private ArrayList<Beverage> createData(){
```

Comment:
We should not return an concrete type here.
`ArrayList` is the concrete type of `List` interface. 
Apply the rule [Program to an interface](http://stackoverflow.com/a/1992479/945808) and keep it as a rule of thumb